packages/@aws-cdk/custom-resources/test/provider-framework/integration/integ.provider.expected.json

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
With the nested stack pattern, it's not possible to have a look here at the resources created by the provider framework...

Reply:
Wow that's right. A bit of a missing capability of our integration test framework. I'll refactor to use sibling stacks to this will be in the expectations.